src/graphorm/index.ts

```diff
@@ -187,6 +187,28 @@ const obj = new GraphORM({
               .andWhere(`${childAlias}."sourceId" = "${parentAlias}".id`),
         },
       },
+      owners: {
+        select: (ctx, alias, qb) => {
+          return qb
+            .select('ARRAY_AGG(sm."userId")')
+            .from(SourceMember, 'sm')
+            .where(`sm."sourceId" = "${alias}".id`)
+            .andWhere(`sm.role = 'owner'`)
```

Comment:
hardcoded `'owner'` since importing `SquadMemberRoles` enum does not resolve correctly. @sshanzel mentioned it is ongoing bundling issue with this project. (same for `'moderator'` below).

Reply:
Not sure did this not work:
```
.andWhere(`sm.role = :role`, { role: SourceMemberRoles.Owner })
```

Reply:
Yes, using the enum was not working. Not sure why it is saying the enum is undefined.

Reply:
@rebelchris @sshanzel looks like `SourceMemberRoles` does work but for some reason when done through `variables` argument it does not work. Refactored to enums here 244f1f61a2d398dd221e1c0c1681af4b3ef10c22, feel free to try